### PR TITLE
Danchadwick wip buff up

### PR DESCRIPTION
**Pull request by DanChadwick**
Update a broad variety of small items, bring other components into harmony with already-updated components. See commit messages for details.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +33,16 @@
 - [#324](https://github.com/miguelcobain/ember-paper/pull/324)
   - paper-button now uses `iconButton` instead of `icon-button`
   - paper-button now uses the action `onClick` instead of `action`
+- [#334](https://github.com/miguelcobain/ember-paper/pull/334)
+  - paper-menu-item now uses action `onClick` instead of `action`.
+  - paper-item now uses action `onClick` instead of `action`.
+  - paper-menu now uses `value` instead of `model`.
+  - paper-checkbox now uses `value` instead of `checked`.
+  - paper-switch now uses `value` instead of `checked`.
+  - paper-select now uses required an `onchange` action.
```
Comment (miguelcobain) [inline CHANGELOG.md]: Should be `onChange`, right?

File: addon/components/paper-item.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ export default Ember.Component.extend(RippleMixin, ProxyMixin, {
 
   hasProxiedComponent: Ember.computed.bool('proxiedComponents.length'),
 
-  hasSecondaryAction: Ember.computed('secondaryItem', 'action', function() {
+  hasSecondaryAction: Ember.computed('secondaryItem', 'onClick', function() {
```
Comment (xomaczar) [inline addon/components/paper-item.js]: Why not destructure computed from Ember, just like in paper-input.js

Comment (DanChadwick) [inline addon/components/paper-item.js]: @xomaczar - Absolutely. The purpose of this PR was to make some API changes. The affected file has not been gone thru and adjusted to conform to the ember-paper style guides (in contributing.md). I only touch paper-item to get rid of all the "action" properties.

Comment (xomaczar) [inline addon/components/paper-item.js]: @DanChadwick makes sense

**Pull request merged**